src/backend/src/lib.rs

```diff
@@ -154,36 +155,63 @@ pub struct Config {
     pub ecdsa_key_name: String,
     // A list of allowed callers to restrict access to endpoints that do not particularly check or use the caller()
     pub allowed_callers: Vec<Principal>,
+    pub supported_credentials: Option<Vec<SupportedCredential>>,
+    /// Root of trust for checking canister signatures.
+    pub ic_root_key_raw: Option<Vec<u8>>,
+}
+
+fn update_config(arg: InitArg) {
```

Comment:
Given that the function does not replace a single fiel but entirely overwrite the config, maybe `init_config` instead of `update_config`?

Reply:
Good point, how about `set_config` then. Init seems to be done only once.